flexinput/src/main/java/com/lytefast/flexinput/fragment/FlexInputFragment.java

```diff
@@ -126,6 +126,9 @@ public void onItemUnselected(final Object item) {
           }
 
           private void updateUi() {
+            if (getView() == null) {
```

Comment:
You def need to grab the view, I meant assigning it to local view (holding on to the view ref) which you did in another commit 👍